app/presenters/finder_content_item_presenter.rb

```diff
@@ -50,6 +50,8 @@ def details
       summary: metadata.fetch("summary", nil),
       facets: schema.fetch("facets", []),
     }
+    details_hash[:default_order] = metadata["default_order"] if metadata["default_order"]
```

Comment:
I just remembered something we said yesterday when Mike thought about filtering out all nil values. A lot of other things are explicitly set to nil in that hash, and we were wondering if filtering them out would change behaviour of other things in other apps that rely or handle those values being nil? See https://github.com/alphagov/specialist-publisher/pull/504#discussion_r33061850.